src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -109,7 +113,8 @@ end
 """
     matrix(A::Array{AbstractAlgebra.Generic.FreeModuleElem{T},1})
 
-Return the matrix whose rows are the vectors in `A`. Of course, vectors in `A` must have the same length and the same base ring.
+Return the matrix whose rows are the vectors in `A`.
+Of course, vectors in `A` must have the same length and the same base ring.
```

Comment:
```suggestion
All vectors in `A` must have the same length and the same base ring.
```

Reply:
That said, it doesn't really matter: one of the next steps after this PR is merged is to go over the code and replace all uses of `submatrix`, `insert_block`, `diagonal_join`, `block_matrix` by their Julia counterparts, and then remove them